logger/module.go

```diff
@@ -92,15 +97,30 @@ func (m *loggerModule) SetBus(bus modules.Bus) {
 
 func (m *loggerModule) GetBus() modules.Bus {
 	if m.bus == nil {
-		m.logger.Fatal().Msg("Bus is not initialized")
+		m.Logger.Fatal().Msg("Bus is not initialized")
 	}
 	return m.bus
 }
 
 func (m *loggerModule) InitLogger() {
```

Comment:
We have a handful of modules calling `CreateLoggerForModule` directly.

Can they not just call `InitLogger` directly?

Reply:
I remember that a couple instances of it would fail due to the lack of a `GetModuleName()` function. I'll double check and if it's indeed possible, substitute all the calls of `CreateLoggerForModule()` with `InitLogger()`

Reply:
I've removed the `InitLogger` in order to leave only the `CreateLoggerForModule`